.github/workflows/terragrunt_plan_staging.yml

```
@@ -51,6 +51,7 @@ env:
   TF_VAR_client_vpn_access_group_id: ${{ secrets.STAGING_CLIENT_VPN_ACCESS_GROUP_ID }}
   TF_VAR_client_vpn_saml_metadata: ${{ secrets.STAGING_CLIENT_VPN_SAML_METADATA }}
   TF_VAR_client_vpn_self_service_saml_metadata: ${{ secrets.STAGING_CLIENT_VPN_SELF_SERVICE_SAML_METADATA }}
+  TF_VAR_gha_arc_pat: ${{ secrets.GHA_ART_PAT }}
```

I love Art too but we might want to be aligned on these variable names.
```suggestion
  TF_VAR_gha_arc_pat: ${{ secrets.GHA_ARC_PAT }}
```